package.json

```diff
@@ -36,14 +36,16 @@
     "build": "if [ \"${KIALI_ENV}\" = \"production\" ]; then npm run build:prod; else npm run build:dev; fi",
     "build-css": "node-sass src/ --output-style compressed --include-path $npm_package_sassIncludes_src -o src/",
     "build:dev": "sh -ac 'source ./.env.upstream; npm run lint && npm run build:kiali'",
-    "build:kiali": "npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) GENERATE_SOURCEMAP=false EXTEND_ESLINT=true react-scripts build",
+    "build:kiali": "npm copy-fonts && npm run copy-img && npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) GENERATE_SOURCEMAP=false EXTEND_ESLINT=true react-scripts build",
     "build:prod": "sh -ac 'source ./.env.downstream; npm run build:kiali'",
+    "copy-fonts": "cp -r node_modules/@patternfly/patternfly/assets/fonts src/",
+    "copy-img": "cp -r node_modules/@patternfly/patternfly/assets/images src/",
     "lint": "eslint --ext js,ts,tsx src",
```

Comment:
Yep, the lines removed were related with PF3:
    "copy-fonts": "cp -r node_modules/patternfly/dist/fonts src/",
    "copy-img": "cp -r node_modules/patternfly/dist/img src/"

Anyway @jshaughn with PF4 we shouldn't add this fonts, components load them